example.cs

```diff
@@ -0,0 +1,119 @@
+using System;
```

Comment:
The location of this file is wrong. It is most certainly not important enough to be at the root of the tree.
Put it in the `scripts` directory